textworld/challenges/spaceship/maker.py

```diff
@@ -0,0 +1,938 @@
+import os
```

Comment:
Is this file still useful? If so, how is it different from `spaceship_game.py`?

Reply:
In my msg to you I have described that this file's gonna be temporally and I will remove it eventually. I just wanna keep it here for a while for ease of access, and I will remove it when the spaceship game design is finalized.

Reply:
Sure. I thought you meant keeping it locally on your computer.